src/common/components/cards/RecentCardsCarousel.js

```diff
@@ -49,13 +65,13 @@ export default class RecentCardsCarousel extends React.Component {
               {breakpoint: 1550, settings: {slidesToShow: 6}},
               {breakpoint: 1750, settings: {slidesToShow: 7}},
               {breakpoint: 1950, settings: {slidesToShow: 8}},
-              {breakpoint: 100000, settings: {slidesToShow: 8}}
+              {breakpoint: 100000, settings: {slidesToShow: this.MAX_CARDS_TO_SHOW}}
```

Comment:
Hmm, if `MAX_CARDS_TO_SHOW` is now a constant, shouldn't this list of breakpoints be dynamically generated based on `MAX_CARDS_TO_SHOW`? 🤔

Reply:
Oh man I don't want to think about that but yeah